code/modules/antagonists/brother/brother.dm

```diff
@@ -86,6 +89,16 @@
 	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] and [key_name_admin(bro)] into blood brothers.")
 	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(bro)] into blood brothers.")
 
+/datum/antagonist/brother/proc/give_pinpointer(var/datum/mind/target_brother)
+	if(owner && owner.current)
+		var/datum/status_effect/agent_pinpointer/brother/P = owner.current.apply_status_effect(/datum/status_effect/agent_pinpointer/brother)
+		P.preset_target = determine_other_brother()
+
+/datum/antagonist/brother/proc/determine_other_brother()
+	var/datum/mind/list/other_brother = team.members - owner
+	for(var/i = 1 to other_brother.len)
+		return other_brother[1]
```

Comment:
as I said before, using a for loop when you're just gonna access the first entry seems redundant, and you might not want to set the pinpointer to point to the one specific brother if there's more than 2